src/libMessage/Messenger.cpp

```diff
@@ -1010,6 +1010,12 @@ bool RepeatableToArray(const T& repeatable, vector<unsigned char>& dst,
   return true;
 }
 
+template <class T>
+void NumberToArray(const T& number, vector<unsigned char>& dst,
+                   const unsigned int offset) {
+  Serializable::SetNumber<T>(dst, offset, number, sizeof(T));
```

Comment:
Oops, `sizeof(T)` won't work for `uint128_t` and `uint256_t`.  You will have to copy the way `NumberToProtobufByteArray` is written.

Reply:
Ok